awx/ui_next/src/components/PaginatedDataList/ToolbarDeleteButton.jsx

```diff
@@ -145,11 +145,16 @@ function ToolbarDeleteButton({
     if (itemsToDelete.some(cannotDelete)) {
       return (
         <div>
-          {errorMessage
-            ? `${errorMessage}: ${itemsUnableToDelete}`
-            : i18n._(
-                t`You do not have permission to delete ${pluralizedItemName}: ${itemsUnableToDelete}`
-              )}
+          {errorMessage ? (
+            <>
+              <span>{errorMessage}</span>
+              <span>{`: ${itemsUnableToDelete}`}</span>
+            </>
+          ) : (
+            i18n._(
```

Comment:
can this just be t`` now?

Reply:
If usage of `t` is not going to completely remove `i18n._(` we will have 2 standards for translation syntax. It there any benefit of one of them?

Reply:
In most cases we will be able to remove `withI18n()` HOC and also i18n that is being injected into the components in favor of simply using `t`